### PR TITLE
Upgraded gradle and substituted previously used features

### PR DESCRIPTION
**Pull request by apoorva-raj**
**Fixes** #2283

**Changes**: 
- <!Removed jCenter() and added mavenCentral(), also updated the versions of libraries used>

**Screen shots for the changes**: 

**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding any value.
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`.
- [x] I have reformatted code and fixed indentation in every file included in this pull request
- [x] My code does not contain any extra lines or extra spaces.
- [x] I have requested reviews from maintainers.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,26 +3,30 @@
 buildscript {
 
     repositories {
-        jcenter()
         google()
-        maven { url "https://plugins.gradle.org/m2/" }
+        mavenCentral()
+        maven { url "https://jitpack.io" }
+        maven{url "https://plugins.gradle.org/m2/"}
```
Comment (CloudyPadmal) [inline build.gradle:9]:
```suggestion
        maven { url "https://plugins.gradle.org/m2/" }
```

File: build.gradle [hunk 2]
```diff
@@ -3,26 +3,30 @@
...
 }
 
+
 allprojects {
     repositories {
-        jcenter()
         google()
+        mavenCentral()
         maven { url "https://jitpack.io" }
+        maven{url "https://plugins.gradle.org/m2/"}
```
Comment (CloudyPadmal) [inline build.gradle:26]:
```suggestion
        maven { url "https://plugins.gradle.org/m2/" }
```

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -44,7 +44,8 @@
         <activity
             android:name=".activity.SplashActivity"
             android:screenOrientation="user"
-            android:theme="@style/AppThemeSplash">
+            android:theme="@style/AppThemeSplash"
+            android:exported="true">
```
Comment (CloudyPadmal) [inline app/src/main/AndroidManifest.xml:48]: What's the purpose of this one?

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (apoorva-raj) [reply app/src/main/AndroidManifest.xml:48]: I added it to resolve an error I encountered after bumping the Android API to 31. You can refer to [b](https://cafonsomota.medium.com/android-12-dont-forget-to-set-android-exported-on-your-activities-services-and-receivers-3bee33f37beb)

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**